packages/config/src/common.ts

```diff
@@ -24,13 +24,39 @@ export type Schema = {
 };
 
 export type Table = {
+  /**
+   * Human-readable label for this table. Used as config keys, library names, and filenames.
+   * Labels are not length constrained like resource names, but special characters should be avoided to be compatible with the filesystem, Solidity compiler, etc.
+   */
   readonly label: string;
+  /**
+   * Human-readable label for this table's namespace. Used for namespace config keys and directory names.
+   */
+  readonly namespaceLabel: string;
+  /**
+   * Table type used in table's resource ID and determines how storage and events are used by this table.
+   */
   readonly type: satisfy<ResourceType, "table" | "offchainTable">;
+  /**
+   * Table namespace used in table's resource ID and determines access control.
+   */
   readonly namespace: string;
-  readonly namespaceLabel: string;
+  /**
+   * Table name used in table's resource ID.
+   */
   readonly name: string;
+  /**
+   * Table's resource ID.
```

Comment:
Like the property descriptions. Just curious is there a reason why it uses single-line comments in `packages/cli/src/deploy/common.ts` but then multi-line comments here, even for single line comments?

Reply:
that file is internal, so not as important to have good docs and those are mostly internal notes

the tsdoc here will get included as part of your IDE tooltips and maybe eventually generated into documentation